swarm/animator.py

```diff
@@ -0,0 +1,42 @@
+#  -*- coding: utf-8 -*-
+__author__ = "Varun Nayyar <nayyarv@gmail.com>"
+
+from matplotlib import pyplot as plt, animation
+import seaborn as sns
+
+plt.rcParams["figure.figsize"] = (14.0, 7.0)
+sns.set()
+
+
+def make_animation(xd, yd, data, title, destfile):
```

Comment:
This is good. maybe destfile could have a default. Later it might be cool to build a title-generator that records some more params, but I really want to build a better plotter (perhaps in. R) that can do some sensible faceting. I'll try to get this working with some output I might log through my own clunky class.

Reply:
Yeah, animator is a low priority fix for me right now. Definitely on my todos